OPAL/br/src/main/scala/org/opalj/br/fpcf/properties/immutability/FieldImmutability.scala

```diff
@@ -65,7 +67,7 @@ case class DependentlyImmutableField(parameter: Set[String]) extends FieldImmuta
             that
         else {
             if (that.isInstanceOf[DependentlyImmutableField])
```

Comment:
1. Don't nest an `if` inside an `else`, use `else if`
2. More importantly: Don't use instanceof, use pattern matching! (In this case, for the whole method)